packages/vant/src/picker/PickerColumn.tsx

```diff
@@ -240,6 +246,7 @@ export default defineComponent({
             option.className,
           ],
           onClick: () => onClickOption(index),
+          onWheel: () => onClickOption(index),
```

Comment:
Why do we need to listen to the wheel event here?

Reply:
@chenjiahan That's about the content of another new PR, there should be a new PR, I'll explain that in the new PR. I submitted the rollback and the current PR is only about the current topic. Could you confirm the current request for this PR?

Reply:
Look good to me~